### PR TITLE
[TextFields] Running clang-format on the elements.

### PR DESCRIPTION
**Pull request by willlarche**
No functional changes.

### CODE REVIEW

File: components/TextFields/src/MDCMultilineTextInputDelegate.h [hunk 1]
```diff
@@ -36,6 +36,6 @@
  UITextView's don't require this method already because they do not have clear buttons. The clear
  button in MDCMultilineTextField is custom.
  */
-- (BOOL)multilineTextFieldShouldClear:(UIView <MDCTextInput>*)textField;
+- (BOOL)multilineTextFieldShouldClear:(UIView<MDCTextInput>*)textField;
```
Comment (romoore) [inline components/TextFields/src/MDCMultilineTextInputDelegate.h]: Nit: Space before asterisk? `(UIView<MDCTextInput> *)textField;`

File: components/TextFields/src/MDCTextField.m [hunk 2]
```diff
@@ -86,7 +86,8 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
     }
 
     [self commonMDCTextFieldInitialization];
-    _cursorColor = [aDecoder decodeObjectForKey:MDCTextFieldCursorColorKey];;
+    _cursorColor = [aDecoder decodeObjectForKey:MDCTextFieldCursorColorKey];
+    ;
```
Comment (romoore) [inline components/TextFields/src/MDCTextField.m]: Nit: Remove extra semicolon.

**Review by romoore — APPROVED** [review]

**Review by ianegordon — APPROVED** [review]
Clang FTW

File: components/TextFields/src/MDCTextField.m (hunk 2 above)
Comment (willlarche) [reply components/TextFields/src/MDCTextField.m]: haha. Oh, clang-format. Thank you for replacing `;;` with `;\n;`.

File: components/TextFields/src/MDCMultilineTextInputDelegate.h (hunk 1 above)
Comment (willlarche) [reply components/TextFields/src/MDCMultilineTextInputDelegate.h]: This is the clang-formatter's chosen format. [shrug]

Comment (willlarche) [reply components/TextFields/src/MDCMultilineTextInputDelegate.h]: Wonder why it did that. Adding space.

**Pull request merged**